packages/trader/src/App/Components/Elements/ContractDrawer/__tests__/contract-drawer-card.spec.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { mockStore } from '@deriv/stores';
+import { ActiveSymbols } from '@deriv/api-types';
+import { getEndTime, isMobile, isDesktop } from '@deriv/shared';
+import ContractDrawerCard from '../contract-drawer-card';
+import TraderProviders from '../../../../../trader-providers';
+
+const mocked_props = {
+    contract_info: {
+        profit: 8.78,
+        validation_error: 'This contract has been sold',
+        shortcode: 'CALL_1HZ100V_19.54_1699708064_10T_S0P_0',
+        underlying: '1HZ100V',
+    },
+    is_accumulator: false,
+    is_collapsed: false,
+    is_market_closed: false,
+    is_mobile: false,
+    is_multiplier: false,
+    is_vanilla: false,
+    is_smarttrader_contract: false,
+    is_sell_requested: false,
+    is_turbos: false,
+    onClickCancel: jest.fn(),
+    onClickSell: jest.fn(),
+    onSwipedUp: jest.fn(),
+    onSwipedDown: jest.fn(),
+    result: 'won',
+    status: 'won',
+    toggleContractAuditDrawer: jest.fn(),
+} as unknown as React.ComponentProps<typeof ContractDrawerCard>;
```

Comment:
why are we making it as unknown first?

Reply:
Because I didn't mocked all props, e.g. here `contract_info` includes only those fields which are used directly in `<ContractDrawerCard />`

Reply:
Can we use `Partial<>` ts utility for it then? - https://www.typescriptlang.org/docs/handbook/utility-types.html

Reply:
Without `unknown`:
<img width="1036" alt="Screenshot 2023-11-14 at 10 32 40" src="https://github.com/binary-com/deriv-app/assets/121025168/eb2cedfc-8d54-499e-8b44-5782e2553391">

Reply:
If I use `Partial<>` I'm still getting error:
<img width="1247" alt="Screenshot 2023-11-14 at 10 59 53" src="https://github.com/binary-com/deriv-app/assets/121025168/7c50b42d-5c7f-43c0-8c27-666f0787c4cd">